Add twig example to every content type and improve content type docs (#566)
sulu/sulu-docs: reference/twig-extensions/functions/sulu_content_load.rst

**Returns**:

.. include:: _page_structure.inc

.. note::

>     The ``sulu_content_load`` twig extension loads the whole page with it content and so
>     can have a negative impact on the performance of the page and if possible should be
>     avoided.

Reviewer: ```suggestion
    The ``sulu_content_load`` twig extension loads and resolves the whole page content.
    This is an expensive operation and can have a negative impact on the performance of 
    the page. If possible, the extension should be avoided.
```
